packages/app-content-pages/src/screens/Team/TeamContainer.js

```diff
@@ -10,7 +10,8 @@ import Team from './Team'
 counterpart.registerTranslations('en', en)
 
 function TeamContainer (props) {
-  const { error, teamData } = props
+  const error = props.error
+  const teamData = props.teamData || [];
```

Comment:
Same comment here about setting defaults in `TeamContainer.defaultProps`.

Reply:
Done - i think this is ready now.